packages/simcore-sdk/tests/unit/test_storage_client.py

```diff
@@ -194,7 +194,7 @@ def storage_v0_service_mock_get_file_meta_data_not_found(
         aioresponses_mocker.get(
             get_file_metadata_pattern,
             status=status.HTTP_200_OK,
-            payload={"error": "No result found", "data": {}},
+            payload={"error": "No result found", "data": None},
```

Comment:
be careful here. I think this test is supposed to simulate what the very old storage service was doing in some legacy services. I do not know if this is still an issue. so I am not sure you can modify this here.

Reply:
Thx. I did not notice that this was a legacy test!